### ISSUE TITLE
Something wrong when installing on Rails3 App

### ISSUE THREAD

**Issue by chorn99** [kaievns/right-rails#19]
I went through the installation exactly as described but I keep getting an error thrown back saying that there is no method of 'tags_field' nor any of the other rightJS methods.

**Comment by kaievns** [comment]
would you kindly post a bit of details, what exactly did you do and how exactly did you try to use it.

btw: it would be a good idea to check https://github.com/MadRabbit/right-rails-demo it has live and working examples of all the widgets

**Comment by chorn99** [comment]
Well, I cloned and ran the demo app and it all worked fine. So then I ran my project again, and I didn't get any errors. I think I just needed to restart the server. However, the functionality isn't working. I am trying to use it in conjunction with ActiveAdmin https://github.com/gregbell/active_admin. However, they're documentation is VERY limited. So I think I just need to look around in the demos to find out what's really going on underneath as well as RTFM. Haha, Thanks for the help!
